pkg/drivers/bess/bess.go

```diff
@@ -590,3 +590,78 @@ func (d *Driver) AddEntryL2FIB(module *network.Switch, macAddr string, gate netw
 	log.Infof("BESS L2 FIB entry created: %+v", entry)
 	return nil
 }
+
+// deleteModule handles deleting a module by name in BESS. Ignores modules that do not exist.
+// name is name of module to delete
+func (d *Driver) deleteModule(name string) error {
+	exists, err := objectExists(d.bessClient, name, MODULE)
+	if err != nil {
+		log.Errorf("Skipping delete for BESS module: %s, error: %v", name, err)
```

Comment:
really worthy of an error? maybe just a warning, even an info since I don't think you would do anything with it since nil is returned - the caller thinks everything is good.

Reply:
Probably just a warning, since the query failed to see if the object existed. I can improve the log message as well.